cmd/snap/cmd_run.go

```diff
@@ -203,6 +203,10 @@ func maybeWaitForSecurityProfileRegeneration(cli *client.Client) error {
 	return fmt.Errorf("timeout waiting for snap system profiles to get updated")
 }
 
+func (x *cmdRun) Usage() string {
+	return "[run-OPTIONS] <NAME-OF-SNAP>.<NAME-OF-APP> [<SNAP-APP-ARG>...]"
```

Comment:
I think the app name should also be optional, right?
  
```suggestion
	return "[run-OPTIONS] <NAME-OF-SNAP>[.<NAME-OF-APP>] [<SNAP-APP-ARG>...]"
```

Reply:
I agree I think this is better to include but we need to make it clear it's optional